README.md

```diff
@@ -6,11 +6,11 @@ This repository contains the [System.Device.Gpio](https://www.nuget.org/packages
 
 The repository also contains [IoT.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings), a growing set of community-maintained [device bindings](src/devices/README.md) for IoT components.
 
-Note: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
+**Note**: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
```

Comment:
perhaps we should generalize this and say something like `This repository is still in experimental stage and all APIs are subject to changes.`. I'd remove mentions about nuget and move it to other documentation page and make the link clearly visible

Reply:
That sounds better.  Will update.